misk/src/test/kotlin/misk/moshi/MoshiModuleTest.kt

```diff
@@ -42,6 +44,24 @@ internal class MoshiModuleTest {
     assertThat(jsonAdapter.fromJson(json)).isEqualTo(value)
   }
 
+  /** Ensure we get our custom Wire Message features, including ignoring unknown fields */
+  @Test
+  fun wireFeaturesAreEnabled() {
+    val jsonAdapter = moshi.adapter<HelloRequest>()
```

Comment:
What are these added tests exercising? They seem to be identical function body and not sure how they relate to the Moshi addLast changes?

Reply:
This test (`wireFeaturesAreEnabled`) is quite similar to `kotlinFeaturesAreEnabled`, but differs in that it's specifically using a wire class, which means it triggers the application of `WireOnlyJsonAdapterFactory`. This isn't strictly related to my PR, it's just a test coverage gap that I'm plugging.

`customAdapterIsPreferredOverWireAdapter` is quite similar to `customAdapterIsPreferred`, but again is using a wire class. This test would fail without my changes (because `WireOnlyJsonAdapterFactory` was taking precedence).

Reply:
Could you update the test name or comments to emphasize what you're specifically exercising?

Reply:
Otherwise it seems unrelated to what it's confirming functionality for

Reply:
Can you suggest what the comments should change to? I'm not sure how else to explain it. Do the [comments on the pre-existing tests](https://github.com/cashapp/misk/blob/1548ad66fa278f67ce466dc4dfee480d63ddb410/misk/src/test/kotlin/misk/moshi/MoshiModuleTest.kt#L26) make sense to you?

Reply:
I'm going to go ahead and merge this so that it doesn't get stale while I'm on PTO, but I'll open a followup PR to make any comment or test method name changes